packages/config-array/src/to-namespaced-path.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @fileoverview Polyfill for Node.js `path.toNamespacedPath`
+ * @author Francesco Trotta
```

Comment:
Just want to double-check: did you write this yourself or is it copied from a polyfill somewhere? (If the latter, we need to include the license.)

Reply:
I wrote it by myself based on the [Node.js implementation](https://github.com/nodejs/node/blob/v22.3.0/lib/path.js#L622-L648). In fact I haven't found any `node:path` polyfill that uses Windows-style paths.

Reply:
Then that means you didn't write it yourself. :) We should include the license from Node.js in this file.

Reply:
Thanks! I've added a link to the Node.js license on GitHub.